static/css/home.css

```diff
@@ -1,3 +1,141 @@
+.container.main-content-shadow {
+    margin-bottom: 0!important;
+    margin: 0;
+    width: 100% !important;
+    max-width: none !important;
+}
+
 .hack-card {
     margin-bottom: 1rem;
-}
\ No newline at end of file
+}
+
+.index-square {
+    min-height: 200px;
+    padding: 5px;
+}
+
+.index-square-content {
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    background-position: center center;
+    background-size: cover;
+    border: 1px solid var(--s-grey);
+}
+
+.index-square-content.more {
+    background-image: initial;
+    background-color: var(--p-navy);
+}
+
+.index-square-content.more .index-square-text {
+    opacity: 1;
+    background: var(--img-highlighter);
+    color: var(--white);
+    width: 100%;
+    padding: 10px;
+    height: 100%;
+}
+
+.index-square-text {
+    opacity: 0;
+}
+
+.index-square-content:hover .index-square-text {
+    opacity: 1;
+    transition: .5s;
+    background: var(--img-highlighter);
+    color: var(--white);
+    width: 100%;
+    padding: 10px;
+    height: 100%;
+}
+
+.index-square-text h1 {
+    position: absolute;
+    bottom: 15px;
+    left: 20px;
+    font-weight: 600;
+    font-size: 1em;
+}
+
+h1.index-square-heading {
+    font-size: 1.2em;
+    padding-left: 0;
+}
+
+h1.index-square-heading.heading-lg {
+    font-size: 1.5em;
+    text-align: center;
+}
+
+.review-slider-separator {
+    width: 200px;
+    height: 3px;
+    color: var(--s-teal);
+    background-color: var(--s-teal);
+}
+
+/* Reviews Slider */
+.reviews {
+    background-color: var(--white);
+}
+
+.row.reviews-slider .prev-step,
+.row.reviews-slider .next-step,
+.row.reviews-slider .reviews-content {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+}
+
+.row.reviews-slider {
+    min-height: 180px;
+    flex-wrap: nowrap;
+}
+
+.row.reviews-slider .reviews-content:not(.active) {
+    display:none;
```

Comment:
Missing _space_ between property and style.

Reply:
Added missing space.